test/helper.rb

```diff
@@ -17,3 +17,7 @@ class Array
     alias_method :sample, :choice
   end
 end
+
+
+# put scmd in test mode
+ENV['SCMD_TEST_MODE'] = '1'
```

Comment:
@jcredding this will go away once Dk auto sets this in the test runner runs.  This is a leaky abstraction - the user shouldn't have to be aware of this implementation detail.